### ISSUE TITLE
Supports DNS Hostname for MQTT server?

### ISSUE THREAD

**Issue by TribuneX** [KmanOz/Sonoff-HomeAssistant#38]
Hi,

does your firmware also allow to specify a DNS hostname for the MQTT server instead of the IP?

**Comment by skoslowski** [comment]
Yes, should work. See e.g. https://github.com/knolleary/pubsubclient/blob/master/examples/mqtt_esp8266/mqtt_esp8266.ino

**Comment by KmanOz** [comment]
Not but I have version 2 that I have been trying to release when I get some time. That will. Hopefully this coming week.
